src/pip/_internal/operations/prepare.py

```diff
@@ -243,15 +243,14 @@ def __init__(  # noqa: PLR0913 (too many parameters)
         lazy_wheel: bool,
         verbosity: int,
         legacy_resolver: bool,
-        resume_retries: int,
     ) -> None:
         super().__init__()
 
         self.src_dir = src_dir
         self.build_dir = build_dir
         self.build_tracker = build_tracker
         self._session = session
-        self._download = Downloader(session, progress_bar, resume_retries)
+        self._download = Downloader(session, progress_bar, session.resume_retries)
```

Comment:
Is there a reason this doesn't go a step further and remove the resume_retries parameter from the Downloader and take the value from the session object?